addons/binding/org.openhab.binding.orvibo/META-INF/MANIFEST.MF

```diff
@@ -12,7 +12,6 @@ Bundle-SymbolicName: org.openhab.binding.orvibo;singleton:=true
 Bundle-Vendor: openHAB
 Bundle-Version: 2.4.0.qualifier
 Import-Package: 
- com.google.common.collect,
```

Comment:
It looks like [s20-sdk-0.0.6-SNAPSHOT.jar uses Guava](https://github.com/tavalin/orvibo-sdk/blob/0df7811510517307b5bcaa55d96247d9ee44e6db/pom.xml#-L68-L72).

Reply:
Okay, that was the POM for 1.1.2-SNAPSHOT, the [0.0.6-SNAPSHOT POM](https://github.com/tavalin/orvibo-sdk/blob/0.0.6/pom.xml) does not seem to use Guava. :upside_down_face: